galaxy.yml

```diff
@@ -3,9 +3,11 @@
 namespace: dzervas
 name: vector
 readme: README.md
+version: 0.1.4a
```

Comment:
I guess you wanted to put this in defaults/main.yml

Reply:
Not really, was just bumping the version. I've reverted it to the original